act-rules-format/act-rules-format.bs

```diff
@@ -433,6 +437,33 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </aside>
 
+<aside class=example>
+  <header>Example subjective applicability for a rule testing that elements styled as a heading use correct heading markup. This rule applicability cannot be written objectively since "styled as a heading" is subjective and depends on the context of the page.</header>
+  <blockquote>
+    <p>The rule applies to any HTML element that is styled as a heading. Elements that are styled as a heading may include features such as a larger font-size than nearby text, bolding or changing of the font, or the use of whitespace or shapes that visually distinguish the element text.</p>
+  </blockquote>
+</aside>
+
+<aside class=example>
+  <header><strong>Non-conforming Example:</strong>This example demonstrates an applicability that attempts to be subjective, but does not meet the "must be unambiguous" requirement. For example, non-text content could refer to images, interactive elements, or even a text emoji like <span role="figure" aria-label="Text smiley face created using a colon and parenthesis">:)</span>. Due to the ambiguity in the applicability it is impossible to tell.</header>
+  <blockquote>
+    <p>Applicability: This rule applies to any non-text content.</p>
+    <p>Expectation: Each test target has an accessible name.</p>
+  </blockquote>
+</aside>
+
+<aside class=example>
+  <header><strong>Non-conforming Example:</strong> While an objective applicability should be used when possible, care should be taken to ensure parts of the applicability are not included in other sections of the rules format such as the expectation. Below is an <strong>incorrect</strong> example of using an objective applicability, where the phrase, ".... except if the test target is part of a text node that is purely decorative or does not express anything in human language" in the expectation should be included in the applicability. When fixed, the full applicability would be, "This rule applies to any visible character in a text node that is a child in the flat tree of an HTML element, except if the test target is part of a text node that is purely decorative or does not express anything in human language."</header>
+  <blockquote>
+    <p>Applicability: This rule applies to any visible character in a text node that is a child in the flat tree of an HTML element </p>
+    <p>Expectation: For each test target, the highest possible contrast between the foreground colors and background colors is at least 3.0:1 for large scale text and 4.5:1 for other texts, except if the test target is part of a text node that is purely decorative or does not express anything in human language.</p>
+  </blockquote>
```

Comment:
**Note:** Additionally, this also makes our published "color contrast" rules not matching the format, so it definitely needs a version bump on the format.
(so that the current rules can still be valid as per format 1.0, until we upgrade them to the newer format)